### PR TITLE
workflow: show latest release in resolve action

### PR DESCRIPTION
**Pull request by dcramer**
#nochanges

### CODE REVIEW

**Comment by dcramer** [comment]
Not sure if this works on mysql (waiting on CI)

**Comment by dcramer** [comment]
Before landing this we need to confirm performance on large accounts

**Comment by dcramer** [comment]
This includes a pretty big refactor of the project details code and serialization to avoid the query cost this would have added.

File: src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx [hunk 1]
```diff
@@ -22,17 +22,13 @@ function getProjectInfoForReview(org) {
...
-          requiresReview++;
-          canReviewAnything = canReviewAnything || canReview;
-        }
         targetList = memberProjects;
       }
       targetList.push({
         projectId: project.id,
         projectName: project.name,
         isMember: team.isMember,
-        requiresReview: !project.callSignReviewed,
```
Comment (dcramer) [inline src/sentry/static/sentry/app/views/requiredAdminActions/setCallsigns.jsx]: we could probably remove all of this review code at this point

**Comment by dcramer** [comment]
@mitsuhiko killed some of the callsign code here

File: src/sentry/api/serializers/models/project.py [hunk 2]
```diff
@@ -170,6 +133,140 @@ def serialize(self, obj, attrs, user):
...
+        'sentry:blacklisted_ips',
+        'feedback:branding',
+        'digests:mail:minimum_delay',
+        'digests:mail:maximum_delay',
+        'mail:subject_prefix',
+        'mail:subject_template',
+    ])
+
+    def get_attrs(self, item_list, user):
+        attrs = super(DetailedProjectSerializer, self).get_attrs(
```
Comment (mitsuhiko) [inline src/sentry/api/serializers/models/project.py]: not sure if this can happen, but if item_list is empty the sql query will have a syntax error.

Comment (dcramer) [inline src/sentry/api/serializers/models/project.py]: i confirmed serializers get skipped if the item_list (or whatever) is falsish

**Review by mattrobenolt — APPROVED** [review]
wcgw

**Review by mitsuhiko — APPROVED** [review]

**Pull request merged**